stdlib/src/base64/base64.mojo

```diff
@@ -21,6 +21,7 @@ from base64 import b64encode
 
 from collections import List
 from sys import simdwidthof
+import bit
```

Comment:
If you'd like, you can use the following references to help your readers.


```suggestion
import bit


 # Wojciech Muła, Daniel Lemire, Base64 encoding and decoding at almost the
 # speed of a memory copy, Software: Practice and Experience 50 (2), 2020.
 # https://arxiv.org/abs/1910.05109
 #
 # Wojciech Muła, Daniel Lemire, Faster Base64 Encoding and Decoding using AVX2
 # Instructions, ACM Transactions on the Web 12 (3), 2018.
 # https://arxiv.org/abs/1704.00605
 
```

Reply:
Done thanks!